Plugins/Public/playercntl_plugin/Message.cpp

```diff
@@ -1241,6 +1245,34 @@ namespace Message
 		return true;
 	}
 
+	bool Message::UserCmd_SetDeathTime(uint iClientID, const wstring &wscCmd, const wstring &wscParam, const wchar_t *usage)
+	{
+		wstring wscParam1 = ToLower(GetParam(wscParam, ' ', 0));
+		bool bShowChatDieTime = false;
+		if (!wscParam1.compare(L"on"))
+			bShowChatDieTime = true;
+		else if (!wscParam1.compare(L"off"))
+			bShowChatDieTime = false;
+		else
+		{
+			PrintUserCmdText(iClientID, L"ERR Invalid parameters");
+			PrintUserCmdText(iClientID, usage);
+		}
+
+		wstring wscCharname = (const wchar_t*)Players.GetActiveCharacterName(iClientID);
+
+		HookExt::IniSetB(iClientID, "msg.chat_dietime", bShowChatDieTime);
+
+		// Update the client cache.
+		map<uint, INFO>::iterator iter = mapInfo.find(iClientID);
+		if (iter != mapInfo.end())
+			iter->second.bShowChatDieTime = bShowChatDieTime;
```

Comment:
Please can we just:
if (mapInfo.find(iClientID) != mapInfo.end())
    mapInfo[iClientID].bShowChatDieTime = bShowChatDieTime;
?

Reply:
I've always been told that when working on a project which other people have started, try to use the same style of operations which the others have been using.

Reference line 1239
```// Update the client cache.
map<uint,INFO>::iterator iter=mapInfo.find(iClientID);
if (iter != mapInfo.end())
	iter->second.bShowChatTime = bShowChatTime;```

This was used in the UserCmd_SetChatTime method.
I'm simply mirroring what it used to complete pretty much the same task.

Reply:
Good answer.